File: dayTwo/Python/Level 4/level_4.py
#Task 1
def sum(numberOne, numberTwo):
    return numberOne + numberTwo

print(sum(4,4))



#Task 2
def evenNumber(number):
    if(number%2 == 0):
        print("Number is even")
    else: print("Number is odd")

print(evenNumber(3))



#Task 3
def square(number):
    return number * number

print(square(4))




#Task 4
def fahrenheit(celcius):
    fahrenheit = (celcius * (9/5)) + 32
    return fahrenheit

print(fahrenheit(8))




#Task 6
def largest(numberOne, numberTwo, numberThree):
    largest = numberOne
    if(numberTwo > numberOne):
        largest = numberTwo
        print(f"Largest is {largest}")
    elif(numberThree > numberTwo):
        largest = numberThree
        print(f"Largest is {largest}")
    else: print(f"Largest is {largest}")

print(largest(87,23,400))




#Task 7
def simpleInterest(principal, rate, term):
    simple_interest = (principal * rate * term)/100
    return simple_interest

print(simpleInterest(10000,10,2))




#Task 8
def areaRectangle(length, width):
    area = length * width
    return area

print(areaRectangle(3,10))



#Task 9
def reverseNumber(number):
    extract = 0
    while(number != 0):
        remainder = number%10
        number = number//10
        extract = extract * 10 + remainder
    print(f"Reversal = {extract} ")
    
print(reverseNumber(567))

File: dayTwo/Java/Level 4/Level4.java
public class Level4{

    public static void main(String[] args){

        System.out.println(sum(4,5));
        System.out.println();

        evenNumber(4);
        System.out.println();

        System.out.println(square(5));
        System.out.println();

        System.out.println(fahrenheit(32));
        System.out.println();

        largest(3,4,2);
        System.out.println();

        System.out.println(simpleInterest(10000,10,2));
        System.out.println();


        System.out.println(areaRectangle(10,2));
        System.out.println();


        System.out.println(reverseNumber(543));
        System.out.println();


    
    }

    //TaskOne    
    public static int sum(int numberOne, int numberTwo){

        return numberOne + numberTwo;
    
    }


    //TaskTwo
    public static void evenNumber(int number){

        if(number%2 == 0){

            System.out.println("Number is even");  
        
        }else{

            System.out.println("Number is odd");

        }

    
    }


    //TaskThree
    public static int square(int number){
    
        return number * number;
    
    }

    //TaskFour
    public static double fahrenheit(double celcius){
    
        double fahrenheit = (celcius * (9/5.0)) + 32;
        return fahrenheit;
    
    }


    //TaskSix
    public static void largest(int numberOne, int numberTwo, int numberThree){
    
        int largest = numberOne;
        if(numberTwo > numberOne ){
    
            largest = numberTwo;
            System.out.printf("Largest is %d", largest);

        }else if(numberThree > numberTwo){

            largest = numberThree;
            System.out.printf("Largest is %d", largest);

        }else{

            System.out.printf("Largest is %d", largest);


        }
    
    }


    //TaskSeven
    public static double simpleInterest(double principal, double rate, double term){

        double simple_interest = (principal * rate * term)/100.0;
        return simple_interest;
    
    }

    //TaskEight
    public static int areaRectangle(int length, int width){

        int area = length * width;
        return area;
    
    }


    //TaskNine
    public static int reverseNumber(int number){

        int extract = 0;
        while(number != 0){

            int remainder = number%10;
            number = number/10;
            extract = extract * 10 + remainder;
        
        }        
        return extract;
    
    }
    



}
